python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py

```diff
@@ -748,11 +748,11 @@ def mutate(self, graphene_info, runId: str):
         return True
 
 
-class GrapheneDagitMutation(graphene.ObjectType):
+class GrapheneWebserverMutation(graphene.ObjectType):
     """The root for all mutations to modify data in your Dagster instance."""
 
     class Meta:
-        name = "DagitMutation"
+        name = "WebserverMutation"
```

Comment:
lets pull these GraphQL schema renames out to reduce risk - they are likely fine but im not 100% certain. We probably just want to call them `Mutation`, `Query`, etc - can discuss that in its own PR

Reply:
done